src/System.Private.ServiceModel/tools/SelfHostedWcfService/packages.config

```diff
@@ -0,0 +1,4 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<packages>
+  <package id="Microsoft.Web.Administration" version="7.0.0.0" targetFramework="net451" userInstalled="true" />
```

Comment:
What is the CRL restful service and where does it live?  Why is it unique to self-host and not IIS host? It's not clear to me why one hosting model has special services the other doesn't have (or why a hosting project has dependencies needed by a service).  Not blocking, but it would reduce questions if we heard a brief plan where this is going.